File: Lesson 29 - JSON/A1.js
// example - callback function 

function sum1(a,b,c){
    console.log(a+b)
    c()
}

function greet(){
    console.log("hello this is callback")
}

sum1(10,20,greet)


// example 2

function sum(a,b){
    console.log(a+b)
}

function calculator(a,b,callback){
    return callback (a, b)
}
calculator(2,3,sum) 

File: Lesson 35/A1.py
def greet(name):
    print("hello",name)

response = input("enter your name")
greet(response)

def numbers(a,b):
    result = a+b
    print(f"the sum of {a} and {b} is {result}")

x = int(input("enter your first number"))
y = int(input("enter your second number"))
numbers(x,y)

def Guess():
    while True:
        guess = int(input("Enter a number from 1 to 10"))
        if guess == 5:
            print("You have guessed the number!!!")
        else:
            print("wrong! try again ")

Guess()